jest.config.js

```diff
@@ -2,5 +2,8 @@ module.exports = {
   moduleFileExtensions: ["js", "json", "jsx", "vue"],
   moduleDirectories: ["node_modules", "src"],
   testEnvironment: "jest-environment-jsdom",
-  setupFilesAfterEnv: ["./setupTests.js"],
+  setupFilesAfterEnv: ["<rootDir>/setupTests.js"],
+  moduleNameMapper: {
+    "\\.(css|scss)$": "<rootDir>/__mocks__/styleMock.js",
```

Comment:
tells jest what to do with imports/requires that are normally handled by webpack